emf/src/main/kotlin/com/strumenta/kolasu/emf/metamodel_building.kt

```diff
@@ -45,7 +45,7 @@ fun EClass.addReference(name: String, type: EClass, min: Int, max: Int): ERefere
     return eReference
 }
 
-fun EClass.addAttribute(name: String, type: EDataType, min: Int, max: Int): EAttribute {
+fun EClass.addAttribute(name: String, type: EClassifier, min: Int, max: Int): EAttribute {
```

Comment:
Why do we want attribute that are not simple data types? Is this to support maps?

Reply:
Yes, here http://wiki.eclipse.org/EMF/FAQ#How_do_I_create_a_Map_in_EMF.3F they recommend to define a Map$Entry subclass with a key and a value attributes, value would be a Node in our case.

Reply:
I think here we can revert back to EDataType

Reply:
@alessiostalla can we revert this part back to EDataType and merge this PR?